runtime/parachains/src/paras_inherent/mod.rs

```diff
@@ -339,7 +339,8 @@ impl<T: Config> Pallet<T> {
 
 		let max_block_weight = <T as frame_system::Config>::BlockWeights::get().max_block;
 
-		METRICS.on_before_filter(candidates_weight + bitfields_weight + disputes_weight);
+		METRICS
+			.on_before_filter((candidates_weight + bitfields_weight + disputes_weight).ref_time());
```

Comment:
these are prometheus metrcis for the runtime, I believe they should continue using u64